app/styles/ilios-common/components/monthly-calendar.scss

```diff
@@ -0,0 +1,100 @@
+.monthly-calendar {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+
+  .month-year {
+    @include ilios-heading-h4;
+    margin: 0 0 .25rem 0;
+    padding: 0;
+  }
+
+  .calendar {
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    grid-gap: 5px;
+    grid-template-rows: 1rem repeat(5, 7rem);
```

Comment:
February 2027 will have a blank row, since all of its days fit within four weeks. I think this edge case is acceptable.

![Selection_024](https://user-images.githubusercontent.com/1410427/73693517-3e6dc080-468b-11ea-85d9-0f02392f2ab6.png)